_config/linktracking.yml

```diff
@@ -4,3 +4,8 @@ Name: cmslinktracking
 SilverStripe\ORM\DataObject:
   extensions:
     - SilverStripe\CMS\Model\SiteTreeLinkTracking
+---
+Name: cmslinktrackingvisibility
+---
+SilverStripe\Security\Member:
+  show_sitetree_link_tracking: true
```

Comment:
Super small thing, could you add a new line please (if you haven't already – you should get https://plugins.jetbrains.com/plugin/7294-editorconfig – that way it'll auto add one)

Reply:
New lines at the end of files is part of PHPStorm by default - just needs to be enabled 🙂 . Settings > Editor > General: 

![image](https://user-images.githubusercontent.com/3260989/45390021-d1aaee00-b671-11e8-9b7b-7606ac535521.png)